linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/tests/logrotate_conf_extra_monthly.fail.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
+# packages: logrotate,crontabs
```

Comment:
I don't think `colon` is the right syntax here:

https://github.com/ComplianceAsCode/content/blob/22ef2f61b24ff4686d745ba68656ca3b88a99e27/tests/ssg_test_suite/rule.py#L311

Reply:
@matejak this should be the only change required here. Can you update the PR?

Reply:
Correct, thanks!